runsc/fsgofer/filter/filter.go

```diff
@@ -31,3 +31,15 @@ func Install() error {
 
 	return seccomp.Install(s)
 }
+
+// InstallUDS installs the standard Gofer seccomp filters along with filters
+// allowing the gofer to connect to a host UDS.
+func InstallUDS() error {
```

Comment:
This name is confusing to me. Maybe it would be clearer to rename it to InstallWithUDS?

I think it might be cleaner to merge this with the regular Install function though.

Here are some ideas about how you could go about doing that:
1. Add a boolean argument to Install.
    ```
    if err := filter.Install(g.hostUDSAllowed); err != nil {
    	Fatalf("installing seccomp filters: %v", err)
    }
    ```
1. Make some kind of object that holds the current table of allowed syscalls.
    ```
    f := NewFilters()
    if g.hostUDSAllowed {
    	f.AllowUDS()
    }
    if err := f.Install(); err != nil {
    	Fatalf("installing seccomp filters: %v", err)
    }
    ```
1. Embrace global state. We are already manipulating the global allowedSyscalls slice's underlying array.
    ```
   func AllowUDS() {
    	allowedSyscalls.Merge(udsSyscalls)
    }
    ```

Reply:
I went the route suggestion 3, and simplified the existing logic as well, directly updating `allowedSyscalls`